src/storage-types/src/sources/mysql.proto

```diff
@@ -15,6 +15,8 @@ import "mysql-util/src/desc.proto";
 
 package mz_storage_types.sources.mysql;
 
+// buf breaking: ignore (compatible change made single -> repeated `initial_gtid_set` field)
```

Comment:
I see this PR removed another ignore from the sinks proto which means that we can forget about them, which is some amount of risk. We can avoid this by introducing a new repeated string field and make it such that we always write the repeated one but we read both and assert that only one of them has data.

Reply:
good idea, switching to that